beacon/sync/src/test/java/tech/pegasys/teku/beacon/sync/forward/multipeer/chains/ThrottlingSyncSourceTest.java

```diff
@@ -101,32 +142,42 @@ void shouldDelayRequestIfBlockLimitAlreadyExceeded() {
     ignoreFuture(source.requestBlocksByRange(UInt64.valueOf(100), count, blocksListener));
 
     // Both requests happen immediately
-    ignoreFuture(verify(delegate).requestBlocksByRange(UInt64.ZERO, count, blocksListener));
+    ignoreFuture(
+        verify(delegate)
+            .requestBlocksByRange(
+                eq(UInt64.ZERO), eq(count), any(RpcResponseListenerWithCount.class)));
     verifyNoMoreInteractions(delegate);
 
     timeProvider.advanceTimeBySeconds(61);
     asyncRunner.executeQueuedActions();
 
-    ignoreFuture(verify(delegate).requestBlocksByRange(UInt64.valueOf(100), count, blocksListener));
+    ignoreFuture(
+        verify(delegate)
+            .requestBlocksByRange(
+                eq(UInt64.valueOf(100)), eq(count), any(RpcResponseListenerWithCount.class)));
   }
 
   @Test
   void shouldDelayRequestIfBlobSidecarsLimitAlreadyExceeded() {
-    final UInt64 count = UInt64.valueOf(MAX_BLOB_SIDECARS_PER_MINUTE);
+    // 17 * 6 = 102 > 100
+    final UInt64 count = UInt64.valueOf(17);
     ignoreFuture(source.requestBlobSidecarsByRange(UInt64.ZERO, count, blobSidecarsListener));
     ignoreFuture(
-        source.requestBlobSidecarsByRange(UInt64.valueOf(100), count, blobSidecarsListener));
+        source.requestBlobSidecarsByRange(UInt64.valueOf(17), count, blobSidecarsListener));
```

Comment:
shouldnt we just re-use count instead of `uint64.valueOf(17)`?

Reply:
yeah, good point, `count` is more cleaner